### PR TITLE
Deploy

### PR DESCRIPTION
**Pull request by erickma1**
A simple page with the name of your app (yes, you need to choose one), and links to the sign up and log in pages.
Sign up and log in pages

The user should be able to register in the app with full name, email and password (all mandatory).
The user can log into the app using email and password.
If the user is not logged in, they can't access pages that require the user to be logged in (all the pages described below).
Home page (categories page)

When the user logs in, they are presented with the categories page.
For each category, the user can see their name, icon and the total amount of all the transactions that belongs to that category.
When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category.
There is a button "add a new category" at the bottom that brings the user to the page to create a new category.
Transactions page

For a given category, the list of transactions is presented, ordered by the most recent.
At the top of the page the user could see the total amount for the category (sum of all of the amounts of the transactions in that category).
There is a button "add a new transaction" at the bottom that brings the user to the page to create a new transaction.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new category" page

The user fills out a form to create a new category, indicating their name and icon (both mandatory).
The user clicks (or taps) the "Save" button to create the new category, and is taken to the home page on success.
When the user clicks on the "Back" button (<), the user navigates to the home page.
"Add a new transaction" page

The user fills out a form to create a new transaction with:
name (mandatory)
amount (mandatory)
categories (mandatory at least one)
The user click (or taps) the "Save" button to create the new transaction, and is taken to the transactions page for that category.
When the user clicks on the "Back" button (<), the user navigates to the transactions page for that category.
Testing requirements
Create unit and integration tests for all the most important components of your RoR application.

### CODE REVIEW

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @erickma1, 

# Suggested changes

so-close-jim-halpert

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Highlights

- No linter errors ✔️
- Descriptive PR ✔️
- Good UI ✔️
- Use partials ✔️
- Correct MVC ✔️

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [ ]  Original design idea by [Gregoire Vella on Behance](https://www.behance.net/gregoirevella). The [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/) requires that you give appropriate credit to the author. Therefore, you must do it in the README of your project.

image


- [ ]  Kindly add the video to your PR for professional documentation 👇🏽

image

- [ ]  Kindly deploy your project to be accessible online and add the link in the corresponding section of the README file, this is a technical requirement for professional documentation. 👇🏽

> The project should be deployed and accessible online.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by rloterh** [comment]
Hi @erickma1 

The _Future Features_ section of your ReadMe is blank. Pls endeavor to add some contents under this section ASAP!

Standing by..Cheers!

**Comment by erickma1** [comment]
> Hi @erickma1
> 
> The _Future Features_ section of your ReadMe is blank. Pls endeavor to add some contents under this section ASAP!
> 
> Standing by..

okay please, doing it now

**Comment by erickma1** [comment]
> > Hi @erickma1
> > The _Future Features_ section of your ReadMe is blank. Pls endeavor to add some contents under this section ASAP!
> > Standing by..
> 
> okay please, doing it now

done please @rloterh

File: README.md [hunk 1]
```diff
@@ -65,6 +65,9 @@
 
 ### Key Features <a name="live-demo"></a>
 
+<a href="https://budget-app-ror2.onrender.com/">Live Demo</a>
+<a href="https://www.loom.com/share/69713d0147f5417f8027599b8a2e802f">Video Presentation</a>
+
```
Comment (rloterh) [inline README.md:70]: - [ ] [OPTIONAL]
Let's make the ReadMe standout even better by
  * including instructions under the _Getting Started_ section, on how to run test cases for the app, for instance, using the command `rspec`.
  * renaming the second the _Key Features_ heading to _Live Demo & Presentation_ or similar, to match the content beneath it. 
 
**Rename the second Key Features:**  
image

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Eric,  

**RoR capstone project** is complete! 🏁 Great job! Congratulations!  🎉💯🌟 

## Highlights 🎯 
🎲 Good job on adding presentation link!
🎲 App deployed correctly!
🎲 Properly added required models, controllers, and corresponding views!
🎲 Schema follow given ERD!
🎲 Nice UI!
🎲 App is running as intended!
🎲 Added required spec test cases and are passing!
🎲 ReadMe looks good!

## Optional suggestions 🧰 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

_Kindly consider my inline comments marked **[OPTIONAL]** under this review_

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**